lib/checks/keyboard/focusable-content.js

```diff
@@ -2,7 +2,7 @@
  * Note:
  * Check if given node contains focusable elements (excluding thyself)
  */
-const tabbableElements = virtualNode.tabbableElements;
+const tabbableElements = virtualNode.tabbableElements();
```

Comment:
This potentially breaks custom rules. I don't think we can change this from being a getter.

Reply:
Is this something we could change in 4.0? As in, could we add back the 2 getters but keep the functions internally, and then we remove the getters in 4.0? Having them on the class itself rather than getters can save quite a bit of memory for larger pages, especially as we potentially add more of these type of functions to the virtual node.